open_gpt/serve/executors/base.py

```diff
@@ -1,5 +1,7 @@
 """The executor wraps the model and provides a simple way to run inference on the model."""
 
+import logging
```

Comment:
why do we need import "loggin" here? We already have defined "logger" in `open_gpt.logs`